Azure/azure-linux/ingest/parser.yml

```diff
@@ -0,0 +1,282 @@
+        output_field: result
+        pattern: >-
+          \(%{USERNAME:user_name}\) CMD
+          \((%{CMD_WITH_PATH}|%{GREEDYDATA:process_args})\)
+        custom_patterns:
+          CMD_WITH_PATH: '%{UNIXPATH:process_executable} %{GREEDYDATA:process_args}'
+  - name: crond
+    description: ''
+    filter: '{{json_event.message.properties.ident == "CROND"}}'
+    external: null
+  - name: grok_sudo
+    description: ''
+    filter: '{{json_event.message.properties.ident == "sudo"}}'
+    external:
+      name: grok.match
+      properties:
+        input_field: '{{json_event.message.properties.Msg}}'
+        output_field: result
+        pattern: '%{SUDO_USER}|%{SUDO_WRONG_PASSWORD}|%{SUDO_CONV_FAILED}'
+        custom_patterns:
+          SUDO_USER: >-
+            pam_%{GREEDYDATA}\(%{GREEDYDATA:action_name}\):
+            %{GREEDYDATA:sudo_observation} for user
+            %{USERNAME:user_name}(%{GREEDYDATA})?
+          SUDO_CONV_FAILED: 'pam_%{GREEDYDATA}\(%{GREEDYDATA:action_name}\): conversation failed'
+          SUDO_WRONG_PASSWORD: >-
+            pam_%{GREEDYDATA}\(%{GREEDYDATA:action_name}\): auth could not
+            identify password for \[%{USERNAME:user_name}\]
+  - name: sudo
+    description: ''
+    filter: '{{json_event.message.properties.ident == "sudo"}}'
+    external: null
+  - name: grok_omsagent
+    description: ''
+    filter: ''
+    external:
+      name: grok.match
+      properties:
+        input_field: '{{json_event.message.properties.Msg}}'
+        output_field: result
+        pattern: 'omsagent : %{GREEDYDATA:omsagent_message}'
+        custom_patterns: {}
+  - name: omsagent
+    description: ''
+    filter: '{{ grok_omsagent.result.omsagent_message != null }}'
+    external:
+      name: kv.parse-kv
+      properties:
+        input_field: '{{ grok_omsagent.result.omsagent_message }}'
+        output_field: result
+        value_sep: '='
+        item_sep: \s;\s
+  - name: grok2_omsagent
+    description: ''
+    filter: '{{ grok_omsagent.result.omsagent_message != null }}'
+    external:
+      name: grok.match
+      properties:
+        input_field: '{{omsagent.result.COMMAND }}'
+        output_field: result
+        pattern: '%{UNIXPATH:process_executable}( %{GREEDYDATA:process_args})?'
+        custom_patterns: {}
+  - name: omsagent2
+    description: ''
+    filter: '{{ grok_omsagent.result.omsagent_message != null }}'
+    external: null
+  - name: action-outcome
+    description: ''
+    filter: >-
+      {{json_event.message.properties.ident == "systemd" or
+      json_event.message.properties.ident == "sudo"}}
+    external: null
+stages:
+  stage2:
+    actions:
+      - set:
+          log.level: '{{json_event.message.level}}'
+          process.pid: '{{json_event.message.properties.pid}}'
+          process.id: '{{json_event.message.properties.pid}}'
+          log.hostname: '{{json_event.message.properties.hostname}}'
+          event.created: '{{json_event.message.time}}'
```

Comment:
This field is generally handled by the log collector (here, `oc-intake-axens`). This is the time at which the event was received. You can probably update the `@timestamp` field to change the date of the event itself.